### ISSUE TITLE
GPA Calculator Enhancements

### ISSUE THREAD

**Issue by Apexal** [Apexal/late#525]
Have it answer questions like "What final grade do I need in [course] to keep my GPA above [threshold]?"

**Comment by robere2** [comment]
Could also do the same for individual courses: "what grade do I need on X exam to keep my course grade above threshold?"

**Comment by cbnt7437** [comment]
Maybe we should make a project for this? Include issue #517

**Comment by Apexal** [comment]
Yes, a project board for this be good since this would be a ~~grand~~large addition that requires changes to the assessment overview page!

**Comment by Apexal** [comment]
I'm going to add minor things to the course grade calculator right now.
